public/js/selfoss-db.js

```diff
@@ -1,6 +1,154 @@
 /**
  * db functions: client data repository
  */
+
+
+selfoss.dbOnline = {
```

Comment:
It makes the base file a little less long and makes sense regarding my work in https://github.com/niol/selfoss/blob/ppr/offline/public/js/selfoss-db.js